<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPEDllCharacteristics</class>
 <widget class="QDialog" name="DialogPEDllCharacteristics">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>279</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>363</width>
    <height>229</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>DLL characteristics</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBoxX" name="checkBoxReadOnly">
       <property name="text">
        <string>Read only</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEditX" name="lineEditDllCharacteristics">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>1</number>
      </property>
      <property name="verticalSpacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBoxX" name="checkBoxHIGH_ENTROPY_VA">
        <property name="text">
         <string notr="true">HIGH_ENTROPY_VA</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBoxX" name="checkBoxNO_BIND">
        <property name="text">
         <string notr="true">NO_BIND</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBoxX" name="checkBoxDYNAMIC_BASE">
        <property name="text">
         <string notr="true">DYNAMIC_BASE</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBoxX" name="checkBoxAPPCONTAINER">
        <property name="text">
         <string notr="true">APPCONTAINER</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBoxX" name="checkBoxFORCE_INTEGRITY">
        <property name="text">
         <string notr="true">FORCE_INTEGRITY</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBoxX" name="checkBoxWDM_DRIVER">
        <property name="text">
         <string notr="true">WDM_DRIVER</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBoxX" name="checkBoxNX_COMPAT">
        <property name="text">
         <string notr="true">NX_COMPAT</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBoxX" name="checkBoxGUARD_CF">
        <property name="text">
         <string notr="true">GUARD_CF</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBoxX" name="checkBoxNO_ISOLATION">
        <property name="text">
         <string notr="true">NO_ISOLATION</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBoxX" name="checkBoxTERMINAL_SERVER_AWARE">
        <property name="text">
         <string notr="true">TERMINAL_SERVER_AWARE</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBoxX" name="checkBoxNO_SEH">
        <property name="text">
         <string notr="true">NO_SEH</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonOK">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonCancel">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonApply">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPushButtonX</class>
   <extends>QPushButton</extends>
   <header>qpushbuttonx.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEditX</class>
   <extends>QLineEdit</extends>
   <header>qlineeditx.h</header>
  </customwidget>
  <customwidget>
   <class>QCheckBoxX</class>
   <extends>QCheckBox</extends>
   <header>qcheckboxx.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
